doc/source/guide-porting.rst

```diff
@@ -370,7 +424,9 @@ correct Python version. For example, instead of ``foo.so``, the shared library
 for the extension module ``foo`` might be named ``foo.cpython-33m.so``.
 
 Your buildsystem might generate these for you already, but if you need to
-modify it, you can get the tags from ``systonfig``::
+modify it, you can get the tags from ``sysconfig``:
+
+.. code-block:: none
```

Comment:
IMO this should be [`pycon`](https://pygments.org/docs/lexers/#pygments.lexers.python.PythonConsoleLexer) (Python console)
```suggestion
.. code-block:: pycon
```